src/gitlab.js

```diff
@@ -32,68 +32,85 @@ export default class VerdaccioGitLab {
 
   }
 
+  static get logPrefix() { return '[gitlab]'; }
+
   authenticate(user: string, password: string, cb: Callback) {
+    this.logger.trace(VerdaccioGitLab.logPrefix, 'authenticate called for user:', user);
 
-    var GitlabAPI = new Gitlab({
+    const GitlabAPI = new Gitlab({
       url:   this.config.url,
       token: password
     });
 
-    GitlabAPI.Users.current().then((response) => {
-      if (user !== response.username) return cb(httperror[403]('wrong gitlab username'));
-      var ownedGroups = [user];
-      GitlabAPI.Groups.all({'owned': 'true'}).then((groups) => {
-        groups.forEach(function(item) {
-          if (item.path === item.full_path) {
-            ownedGroups.push(item.path);
+    GitlabAPI.Users.current().then(response => {
+      if (user !== response.username) { return cb(httperror[401]('wrong gitlab username')); }
```

Comment:
This needs to return a 401 error, or the npm client won't show a login error

Reply:
nice!